crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -105,15 +169,43 @@ pub struct StandardMaterial {
 
     /// Support two-sided lighting by automatically flipping the normals for "back" faces
     /// within the PBR lighting shader.
-    /// Defaults to false.
+    ///
+    /// Defaults to `false`.
     /// This does not automatically configure backface culling, which can be done via
     /// `cull_mode`.
     pub double_sided: bool,
-    /// Whether to cull the "front", "back" or neither side of a mesh
-    /// defaults to `Face::Back`
+
+    /// Whether to cull the "front", "back" or neither side of a mesh if set to `None`.
```

Comment:
Bevy uses a counter-clockwise winding convention.
```suggestion
    /// Whether to cull the "front", "back" or neither side of a mesh if set to `None`.
    ///
    /// The front face is defined by the primitive vertices winding in a counter-clockwise
    /// direction. This means that if the vertices appear to wind around in a counter-clockwise
    /// direction from the viewer's position, then the front face is being observed. If they wind
    /// in a clockwise direction then the back face is being observed.
```

Reply:
Good idea! Though I'll edit the suggestion a bit first. It's a bit… winding :wink: